ovis_moveit_config/package.xml

```diff
@@ -5,8 +5,8 @@
   <description>
      An automatically generated package with all the configuration and launch files for using the ovis with the MoveIt! Motion Planning Framework
   </description>
-  <author email="capra@clubcapra.com">Club Capra</author>
-  <maintainer email="capra@clubcapra.com">Club Capra</maintainer>
+  <author email="alexandre.lapointe.4@ens.etsmtl.ca">Alexandre Lapointe</author>
+  <maintainer email="alexandre.lapointe.4@ens.etsmtl.ca">Alexandre Lapointe</maintainer>
```

Comment:
Super ignorable nitpick: the club's email or our personal ones should outlive our institutional one. I don't mind if you use your own, but I doubt you will read the ÉTS one in a couple years, assuming they don't deactivate it.